### PR TITLE
Reformat sourcecode, add missing return type hints, add missing PHPDoc comments

### PR DESCRIPTION
**Pull request by wazum**
based on PSR-12/Symfony guidelines

### CODE REVIEW

**Comment by benr77** [comment]
Thanks for the PR!

Happy to merge PSR-12 fixes etc as this is much needed. However, I don't want to litter the code with pointless phpdocs - can you remove the @param and @return declarations where it's already specified in the method signature or return type-hints please.

File: DependencyInjection/HeadsnetDomainEventsExtension.php [hunk 1]
```diff
@@ -17,9 +17,14 @@
...
 class HeadsnetDomainEventsExtension extends Extension
 {
-	public function load(array $configs, ContainerBuilder $container)
-	{
-		$loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
-		$loader->load('services.xml');
-	}
+    /**
+     * @param array $configs
+     * @param ContainerBuilder $container
```
Comment (benr77) [inline DependencyInjection/HeadsnetDomainEventsExtension.php:22]: This is redundant as it's specified in the method signature.

File: Doctrine/DBAL/Types/DateTimeImmutableMicrosecondsType.php [hunk 2]
```diff
@@ -20,22 +20,32 @@
 
 class DateTimeImmutableMicrosecondsType extends VarDateTimeImmutableType
 {
-    public function getName()
+    /**
+     * @return string
```
Comment (benr77) [inline Doctrine/DBAL/Types/DateTimeImmutableMicrosecondsType.php:24]: Again, the entire docblock is redundant as string is already specified in the method return type-hint

**Comment by benr77** [comment]
Thank you for the changes. I will tag a release too.

**Comment by wazum** [comment]
Thanks for the nice package :-)

**Pull request merged**